xapi-xenopsd-simulator.opam

```diff
@@ -0,0 +1,11 @@
+opam-version: "1.2"
+name: xapi-xenopsd-simulator
```

Comment:
`opam lint "xapi-xenopsd-simulator.opam"` does not pass: this string should be quoted.